docs/TuxActionStores.md

```diff
@@ -0,0 +1,170 @@
+
+Well, with ActionStores, we provide an alternative approach to registering your stores with your actions.
+
+## <a id="Premise"></a>Premise [#](#Premise)
+>Tux `ActionStores` is a module providing the interface for creating stores that are pre-packaged with a 'register' convenience method for automatically registering the store with actions.
+
+## <a id="Implementation"></a>Implementation [#](#Implementation)
+### <a id="Requiring-ActionStores"></a>Requiring ActionStores [#](#Requiring-Action-Stores)
+
+The interface for creating `ActionStores` is exposed via a `createStore` constructor function on the `ActionStores` object.
+
+```
+    var ActionStores = require('tux/Stores/ActionStores');
+```
+
+### <a id="Creating-Action-Stores"></a>Creating ActionStores [#](#Creating-Action-Stores)
+`ActionStores` are an implementation on top of the basic `TuxStore`. Thus, they have all of the same properties. The only difference is that `ActionStores` expect to have a `register` method passed in. If this `register` key is defined, then the ActionStore will invoke the method with the store as the context and use the object the method returns to register the store with `TuxActions`. Let's take a look at an example
+
+```
+    var actionStore = ActionStores.createStore({
+      _messages: [],
+
+      onCreate: function (message) {
+        this._messages.push(message);
+        this.emitChange();
+      },
+
+      register: function () {
+        return {
+          messages: {
+            create: this.onCreate
+          }
+        }
+      }
+    });
+```
+
+Below is the API documentation for this method.
+### <a id="ActionStores-createActionStore"></a>ActionStores.createStore [#](#ActionStores-createStore)
+Returns an `actionStore` instance based on the passed in `storeProps`.
+
+```
+    var actionStore = ActionStores.createStore(storeProps);
+```
+
+#### <a id="storeProps"></a>Parameter - `storeProps` - type: OBJECT - required [#](#storeProps)
+StoreProps is an object containing actions that will encapsulate the store's data, with an optional register method which, if provided, will automatically register all the actions with the newly created store. Let's see what this looks like:
+
+```
+    var messageStoreProperties = {
+      _messages: [],
+
+      onCreate: function (message) {
+        this._messages.push(message);
+        this.emitChange();
+      },
+
+      register: function () {
+        return {
+          messages: {
+            create: this.onCreate
+          }
+        }
+      }
+    };
+
+```
+
+#### <a id="storeProps-register"></a>Parameter Method - `storeProps.register` - type: FUNCTION - optional [#](#createActionStore)
+The `register` method is an optional key on the `storeProps` parameter which, if defined, will invoke `Actions.register`, passing in the store as the first argument and the object returned by this `storeProps.register` method as the second argument. A quick reference to what `Actions.register` looks like:
```

Comment:
Change last sentence to: Here is what the `register` method performs on the backend.